android/buildSrc/src/main/kotlin/com/badoo/ribs/templategenerator/TemplateGeneratorPlugin.kt

```diff
@@ -42,7 +42,8 @@ class TemplateGeneratorPlugin : Plugin<Project> {
                 name = it.name ?: throw IllegalArgumentException("Template name is mandatory"),
                 fromProject = it.fromProject
                     ?: throw IllegalArgumentException("fromProject property is mandatory"),
-                modulePackage = it.fromProject!!.androidPackageName,
+                modulePackage = it.fromProject!!.androidPackageName ?: it.modulePackage
```

Comment:
`androidExtension.defaultConfig.applicationId` doesn't work for library modules. didn't find any easy solution to get the package name from the manifest, so I've just added a parameter to gradle extension